src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1173,6 +1183,32 @@ private string GetFetchLogicForRuntimeSpecifiedCustomConverter()
 }}";
             }
 
+            private string GetFetchLogicForGetCustomConverter_PropertiesWithFactories()
+            {
+                return @$"
+
+private {JsonConverterTypeRef}<T> {GetConverterFromFactoryMethodName}<T>({JsonConverterFactoryTypeRef} factory)
```

Comment:
Where is this method being invoked? I searched the diff for other instances of `GetConverterFromFactoryMethodName` but couldn't find any.

Reply:
It is used when creating the "converter" metadata. In the PR description, search for `converter: jsonContext.GetConverterFromFactory`